packages/sui-test/bin/sui-test-browser.js

```diff
@@ -10,31 +10,32 @@ const { cleanStack } = require('./karma/util')
 program
   .option('-W, --watch', 'Run in watch mode')
   .option('-C, --ci', 'Run a Firefox headless for CI testing')
+  .option('-P, --pattern <pattern>', 'Path pattern to include', 'test/**/*Spec.js')
+  .option('-I, --ignore-pattern <ignorePattern>', 'Path pattern to ignore for testing', false)
   .on('--help', () => {
     console.log('  Description:')
     console.log('')
-    console.log('  Run tests in chorme')
+    console.log('  Run tests in Chrome')
     console.log('')
     console.log('  Examples:')
     console.log('')
     console.log('    $ sui-test browser')
     console.log('')
   })
   .parse(process.argv)
-;(() => {
-  const { watch, ci } = program
-  runner({ watch, ci })
-    .then(output => {
-      if (output != null) process.stdout.write(output + '\n')
-      if (!watch) process.exit(0)
-    })
-    .catch(err => {
-      if (!(typeof err.code === 'number' && err.code >= 0 && err.code < 10)) {
-        process.stderr.write(
-          chalk.red(cleanStack((err && (err.stack || err.message)) || err)) +
-            '\n'
-        )
-      }
-      process.exit(err.code || 1)
-    })
-})()
+
+const { watch, ci, pattern, ignorePattern } = program
```

Comment:
We can use this pattern `/test/**/*Spec.js` like a default pattern to avoid to force the user to use always a pattern flag.

Reply:
That's the default pattern. `/test/**/*Spec.js`

https://github.com/SUI-Components/sui/pull/231/files#diff-0326ee9225e945d3c1b4894c5d101c9eR13

But now, default values are in the commander, so the user will be informed with the default path used. :)